install-nhc.sh

```diff
@@ -85,9 +85,19 @@ fi
 # Check cmake version + install if necessary
 output=$(cmake --version | sed -n 1p | sed 's/[^0-9]*//g')
 export NHC_CMAKE=cmake
+# Find VM OS
```

Comment:
Lets not do lines 88-93 and lines 97-100 . You can determine the OS by looking at this file: /etc/os-release. 

Please revert this change.

Reply:
Do we want to determine the OS using the other option in order to install Boost? Or remove finding the OS altogether?